src/Nuget/Microsoft.VisualStudio.AppDesigner/Microsoft.VisualStudio.AppDesigner.Nuspec

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.VisualStudio.AppDesigner</id>
+    <summary>AppDesigner implementation for Project Properties</summary>
+    <description>
+      This is the AppDesigner implementation of the Project properties page in VisualStudio
```

Comment:
This is unclear. What's an appdesigner implementation of something? Maybe say something like "This package implements the AppDesigner, which is the designer host used for project property pages among other things in Visual Studio"